less/mosaico/static-pages/kundservice.less

```diff
@@ -0,0 +1,35 @@
+@import (reference) "../../ksf-colors";
+
+.static-page__customer-service {
+  display: flex;
+  flex-direction: column;
+
+  .customer-service__headline {
+    margin: 20px 0;
+    font-size: 2.2rem;
+    line-height: 2.2rem;
```

Comment:
This could maybe be `line-height: 1;` to keep it automatically tied to the `font-size`? (Looks like it would then be 2.2rem * 1 according to here: https://developer.mozilla.org/en-US/docs/Web/CSS/line-height)